src/envoy/mixer/http_control.cc

```diff
@@ -124,6 +125,12 @@ void FillRequestInfoAttributes(const AccessLog::RequestInfo& info,
     attr->attributes[kAttrNameResponseTime] =
         Int64Value(info.duration().count());
   }
+  if (info.responseCode().valid()) {
+    attr->attributes[kStatusCode] =
+        StringValue(std::to_string(info.responseCode().value()));
+  } else {
+    attr->attributes[kStatusCode] = StringValue("500");
```

Comment:
`responseCode()` may not cover all the cases (it is `Optional`). Do we need to check all the possible [ResponseFlags](https://github.com/lyft/envoy/blob/453b51138c5e23e2072297b6a2cb85d74c4d39c7/include/envoy/http/access_log.h#L13)? And what should be the corresponding status code?